cmd/neofs-adm/internal/modules/config/config.go

```diff
@@ -0,0 +1,124 @@
+package config
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"text/template"
+
+	"github.com/nspcc-dev/neofs-node/pkg/innerring"
+	"github.com/spf13/cobra"
+)
+
+type configTemplate struct {
+	Endpoint      string
+	AlphabetDir   string
+	MaxObjectSize int
+	EpochDuration int
+	Glagolitics   []string
+}
+
+const configTxtTemplate = `rpc-endpoint: {{ .Endpoint}}
+alphabet-wallets: {{ .AlphabetDir}}
+network:
+  max_object_size: {{ .MaxObjectSize}}
+  epoch_duration: {{ .EpochDuration}}
+# if credentials section is omitted, then neofs-adm will require manual password input
+credentials:{{ range.Glagolitics}}
+  {{.}}: password{{end}}
```

Comment:
What's the command for generating empty section?

Reply:
There is no such command. Default config will always have `credential` section with seven alphabet keys.
Administrator is supposed to change this config later and this section can be removed by administrator (that what the comment above for).